src/NzbDrone.Mono/ProcMountProvider.cs

```diff
@@ -17,23 +18,26 @@ public class ProcMountProvider : IProcMountProvider
     {
         private static string[] _fixedTypes = new [] { "ext3", "ext2", "ext4", "vfat", "fuseblk", "xfs", "jfs", "msdos", "ntfs", "minix", "hfs", "hfsplus", "qnx4", "ufs", "btrfs" };
         private static string[] _networkDriveTypes = new [] { "cifs", "nfs", "nfs4", "nfsd", "sshfs" };
+        private static readonly Regex OctalRegex = new Regex(@"\\\d{3}", RegexOptions.Compiled);
 
         private static Dictionary<string, bool> _fileSystems;
 
+        private readonly IDiskProvider _diskProvider;
         private readonly Logger _logger;
 
-        public ProcMountProvider(Logger logger)
+        public ProcMountProvider(IDiskProvider diskProvider, Logger logger)
```

Comment:
After we talked I looked at moving `GetMounts` to its own interface, but `GetMounts` is used in a number of places in DiskProvider, so I don't think its worth the effort of unwinding it all. I didn't see this circular dependency and it managed to pass all the tests on TeamCity, maybe we can rename this to something that indicates its a composition in a better way.